debian-9/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:10.10.0-alpine
```

Comment:
will that become Oryx image somehow? Shouldn't we use a placeholder?

Reply:
this is a place holder